dex/src/main/scala/com/wavesplatform/dex/api/websockets/WsServerMessage.scala

```diff
@@ -0,0 +1,31 @@
+package com.wavesplatform.dex.api.websockets
+
+import play.api.libs.json._
+
+trait WsServerMessage extends WsMessage
+
+object WsServerMessage {
+  // Will be never propagated to a client
+  case object Complete extends WsServerMessage {
+    override val tpe: String = "unused"
+  }
+
+  // TODO play-json discriminator
```

Comment:
Guess its a way to make this code (and `WsClientMessage`) succinct

Reply:
A some research is required to do this :)